src/wasm/wasm-binary.cpp

```diff
@@ -1511,23 +1524,27 @@ void WasmBinaryBuilder::readTableElements() {
 
 void WasmBinaryBuilder::readNames() {
   if (debug) std::cerr << "== readNames" << std::endl;
+  auto name_type = getU32LEB();
+  auto subsection_size = getU32LEB();
+  WASM_UNUSED(subsection_size);
```

Comment:
Oh, heh, nice :)

Btw, is the other code you mention your gcc wasm backend? Are you connecting those in some way, or just using binaryen to inspect the output from gcc?